### PR TITLE
Reuse ClientConn in ipsec

### PR DESCRIPTION
**Pull request by tpantelis**
There are several funtions that obtain a ClientConn via getClient
locally which leads to multiple ClientConn instances created and
closed in a call chain. Optimize by creating a single ClientConn and
passing to the lower-level functions.

Signed-off-by: Tom Pantelis <tompantelis@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 submariner-engine
 .dapper
 bin
+.project
```
Comment (skitt) [inline .gitignore]: Perhaps this should go in a separate patch (dedicated to ignoring Eclipse files). You can also configure git globally to ignore files specific to your IDE ;-).

**Comment by tpantelis** [comment]
I removed the changes to .gitignore - will push another patch/PR for that.

**Comment by Oats87** [comment]
Thank you for the PR. We will review this and get this merged ASAP.

**Comment by mangelajo** [comment]
Thank you @tpantelis the changes make sense, and it compiles, I'm merging it.
